CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+## v1.1.0.0 - "Moore Features"
+
+The most notable addition in this release is the implementation of the Boyer-Moore string search algorithm.
+
+- Added `Data.Text.AhoCorasick.Splitter` for splitting a lot of text using the same needle
+- Added `Data.Text.BoyerMoore.Automaton`, a UTF-16 implementation of Boyer-Moore
+- Added `Data.Text.BoyerMoore.Searcher` for searching for multiple needles at once using Boyer-Moore
+- Added `Data.Text.BoyerMoore.Replacer` for replacing text based on the Boyer-Moore search
+- Added optional `FromJSON`/`ToJSON` instances for most types (can be toggled via `aeson` cabal flag)
+- Extracted the UTF-16 manipulation functions from `Data.Text.AhoCorasick.Automaton` into `Data.Text.Utf16`
+- Changed `Data.Text.AhoCorasick.Searcher.Searcher` to remember the case sensitivity used for constructing the searcher
```

Comment:
I also find it useful when changelogs put breaking changes up front and clearly marked as such, because usually when updating dependencies, I just get the new version because a newer version happened to be published, and then the only thing I am interested in is whether I need to take any further action besides bumping the version number. If you have to read through a list of dozens of bullet points, that becomes very difficult.

I usually write something like

> **Compatibility**:
> * Removed x, use y instead
> * Changed z
>
> Other changes:
> * Added p
> * Improved q

Reply:
I split the list of changes now.